### ISSUE TITLE
SVM 

### ISSUE THREAD

**Issue by YHuixia** [kshiprakode/SVM-Machine-Learning#1]
hello,My name is Sun shine.I have read you code,I think it is very helpful for me,but  having a question in train_data.m file.
if(value < 1)
                   W = W - learning_rate * ( W - C * label * images_train(i,:));
                   b = b - learning_rate * ( b - C * label);
            else
                W = W - learning_rate * W;
                b = b - learning_rate * b;
what dos C mean?in this formula,
